pallet/staking/src/lib.rs

```diff
@@ -940,6 +930,7 @@ pub use pallet::*;
 
 type RewardPoint = u32;
 type Power = u32;
+type Vote = u32;
```

Comment:
Can you add some comment on the differences between the vote and power here? I am not sure if the vote equals to the  power * adjustment.